packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -683,11 +685,14 @@ class MemoryBodyState extends State<MemoryBody>
       rowChildren.addAll(legendPart(name2, image2, 20.0));
     }
 
-    return Container(
-        padding: const EdgeInsets.fromLTRB(10, 0, 0, 2),
-        child: Row(
-          children: rowChildren,
-        ));
+    return Expanded(
+      child: Container(
+          padding: const EdgeInsets.fromLTRB(10, 0, 0, 2),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: rowChildren,
+          )),
```

Comment:
nit: not your fault, but please add a trailing comma between these two parens here

Reply:
Thanks, fixed!